spec/acceptance/basic_spec.rb

```diff
@@ -79,6 +79,11 @@ class { 'pulpcore':
     its(:exit_status) { is_expected.to eq 0 }
   end
 
+  describe command("PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager diffsettings") do
+    its(:stdout) { is_expected.to match(/^WORKER_TTL = 30/) }
```

Comment:
Can we move this in the `diffsettings` command above? That makes the execution faster since it doesn't need to call `diffsettings` multiple times.